manifests/params.pp

```diff
@@ -11,7 +11,7 @@
   $repo_proxy_password         = 'absent'
   $repo_proxy_username         = 'absent'
   $repo_release                = undef
-  $repo_url_suffix             = '12.x'
+  $repo_url_suffix             = '18.x'
```

Comment:
Is that a breaking change?

Reply:
(I'll probably split it out to an own PR, once I verified it fixes what I think it should fix on D12/U22)